docker-compose.yml

```diff
@@ -2,13 +2,35 @@ version: "3.1"
 services:
   shock:
     image: kbase/kb_shock:${TRAVIS_BRANCH:-latest}
+    command:
+      - "-template"
+      - "/kb/deployment/conf/.templates/shock-server.cfg.templ:/kb/deployment/conf/shock-server.cfg"
+      - "-wait"
+      - "tcp://mongo:27017"
+      - "-wait"
+      - "tcp://mongoinit:8080"
+      - "-timeout"
+      - "150s"
+      - "/kb/deployment/bin/shock-server"
+      - "--conf"
+      - "/kb/deployment/conf/shock-server.cfg"
     ports:
       - "7044:7044"
+    depends_on: ['mongoinit',"mongo"]
   mongoinit:
-    image: mongo:2
+    image: kbase/db_initialize:latest
```

Comment:
@kkellerlbl This is a new image that specializes in loading databases and has a little wrapper script that brings up a netcat listener after loading the data, which a dependent container can check as a sign that the data load completed.

Reply:
We may want to tell people who want to preserve an existing environment (either CI or maybe someone wants their local minikb persistent) not to use the mongoinit container after the initial init.